Content.Shared/Body/Events/MetabolismExclusionEvent.cs

```diff
@@ -0,0 +1,6 @@
+using Content.Shared.Chemistry.Reagent;
+
+namespace Content.Shared.Body.Events;
+
+[ByRefEvent]
+public readonly record struct MetabolismExclusionEvent(List<ReagentQuantity> ReagentList);
```

Comment:
needs a doc comment explaining what this does, what it's raised on by what & for what purposes, etc.